components/header/index.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import './index.css'
+import Logo from '../logo'
+import Header from '../../design/header'
+import Avatar from '../../design/avatar'
+import SearchBar from '../searchbar'
+
+export default React.memo(({ className }) => {
+  return (
+    <Header className={className}>
+      <Header.Item className="header-area-brand">
```

Comment:
Why do you need `Header.Item`?

Reply:
Just to be consistent. I wanted to have it like only HederItem can be direct child of the Header component.